lists/eg.csv

```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
+https://bedayaalgbt.wordpress.com/,LGBT,LGBT,2018-02-10,OONI,
+https://solidaritywithegyptlgbt.wordpress.com/,LGBT,LGBT,2018-02-10,OONI,
+https://alborsanews.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://elbadil.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://www.aljazeera.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+http://www.watantvlive.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://www.al-sharq.com/,NEWS,News Media,2018-02-10,OONI,
+https://www.madamasr.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://www.sasapost.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://shabab6april.wordpress.com/about/shabab-6-april-youth-movement-about-us-in-english/,POLR,Political Criticism,2018-02-10,OONI,
+https://6april.org/,POLR,Political Criticism,2018-02-10,OONI,
+http://www.turkpress.co/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://arab-turkey.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://www.akhbarturkiya.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://manzalawy.net/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://thawrah2day.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://www.akhbaralsabah.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://alamatonline.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://www.aljisr-news.com/news/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://m.alkessa.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://almanassa.com/ar,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://www.amlalommah.net/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://www.ansarportsaed.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://arabi21.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://arabsolaa.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://ar.qantara.de/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://asrararabiya.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://www.dailysabah.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://www.dakahliaikhwan.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://egyptdailynews.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://egyptwindow.net/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://elbehira.net/elbehira/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://elwehda.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://fj-p.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://gharbiaonline.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://cluw.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+http://anhri.net/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+https://www.hrw.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
```

Comment:
This is already present in the global test list and some paths of the https site are already present in the EG list. It should be dropped.